src/conversation/conversation.service.ts

```diff
@@ -110,16 +111,31 @@ export class ConversationService {
 
     const { message } = continueConversationDto;
 
+    // *Check if the user wants to restart the conversation
+    if (message.trim() === 'restart') {
```

Comment:
I didn't know this feature existed in Bloom, but if it does, we don't want it! It does not make any sense to "reset" a conversation!!

Reply:
They mentioned in the welcome_message "If I'm off track, just say so! If you'd like to reset our dialogue, use the /restart command.", since they are designed to be on Discord, a reset button should be present, I'm just trying to mimic if a similar action can be implemented. So I added this feature, and if this feature is not necessary, then I can remove this feature.